test-pnp.sh

```diff
@@ -21,15 +22,15 @@ do
   cd $initialDir
 
   echo "--- Testing $testCase ---"
-  mkdir -p "./e2e-tests/$testCase"
-  cp -r "./examples/$testCase/." "./e2e-tests/$testCase"
-  cd "./e2e-tests/$testCase"
+  mkdir -p "/tmp/e2e-tests/$testCase"
```

Comment:
Wont work for Windows, use something like `mktemp` to get a temp folder

Reply:
@merceyz changed to use `mktemp -d`